span_context.go

```diff
@@ -344,9 +344,9 @@ func (c *SpanContext) isDebugIDContainerOnly() bool {
 
 func (t TraceID) String() string {
 	if t.High == 0 {
-		return fmt.Sprintf("%x", t.Low)
+		return fmt.Sprintf("%016x", t.Low)
```

Comment:
I would also do it in L382 for span ID

Reply:
Done.

Should I squash? Or will you do that on pre-merge?

Reply:
we squash on merge